odpf/optimus/runtime_service.proto

```diff
@@ -554,6 +554,8 @@ message RegisterSecretRequest {
     string project_name = 1;
     string secret_name = 2;
     string value = 3; // base64 encoded secret value
+    string namespace_name = 4;
+    bool updateOnly = 5;
```

Comment:
this should not be camel case

Reply:
Thank you for pointing out. I will make this change.

Reply:
Marking as resolved, changes already done.